omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java

```diff
@@ -63,6 +64,9 @@ public class FhirRestServlet extends RestfulServer implements ModuleLifecycleLis
 	private static final List<String> DEFAULT_NARRATIVE_FILES = Arrays.asList(FhirConstants.OPENMRS_NARRATIVES_PROPERTY_FILE,
 	    FhirConstants.HAPI_NARRATIVES_PROPERTY_FILE);
 	
+	@Autowired
+	private EpisodeOfCareFhirResourceProvider dao;
```

Comment:
You shouldn't need to autowire this here. (Also the name of the variable is slightly confusing.

Reply:
Removed, this was an artifact left over when I was debugging something